temporal-sdk/src/main/java/io/temporal/worker/WorkflowImplementationOptions.java

```diff
@@ -64,27 +70,70 @@ public Builder setFailWorkflowExceptionTypes(
       return this;
     }
 
+    /**
+     * Set individual activity options per activityType.
```

Comment:
Add information about how this relates to the setActivityOptions when creating a stub.